### PR TITLE
Consistent and unique container names

### PR DESCRIPTION
**Pull request by aliok**
Cherry pick and modify https://github.com/knative/eventing/issues/2794

@matzew I just created the PR against 0.13.0 branch. Should we create a new branch? Don't know if our CI is able to mutate the artifacts after they're released - ideally it shouldn't.

### CODE REVIEW

File: openshift/release/knative-eventing-v0.13.0.yaml [hunk 1]
```diff
@@ -2262,7 +2262,7 @@ spec:
     spec:
       serviceAccountName: eventing-controller
       containers:
-      - name: eventing-controller
+      - name: channel-broker-controller
```
Comment (matzew) [inline openshift/release/knative-eventing-v0.13.0.yaml]: you "change" this file, by running the below, right ?

```
make RELEASE=v0.13.0 generate-release 
```

At least I do not get a _diff_ on the file

Comment (aliok) [reply openshift/release/knative-eventing-v0.13.0.yaml]: kind of...

**Comment by matzew** [comment]
> I just created the PR against 0.13.0 branch. Should we create a new branch? Don't know if our CI is able to mutate the artifacts after they're released - ideally it shouldn't.

the image builds are different from the actual name of the container, im the k8s yaml.

image builds, are based on path to our `ci-operator` folder, and config for that is here:

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.13.0__44.yaml#L55-L62

**Pull request merged**